packages/flutter_image/test/network_test_server.dart

```diff
@@ -15,9 +15,13 @@ Future<void> main() async {
       request.response.add(_kTransparentImage);
     } else if (request.uri.path.endsWith('/error.png')) {
       request.response.statusCode = 500;
+    } else if (request.headers.value('ExtraHeader') == 'special' &&
```

Comment:
Please reverse the order of the checks here to make the intent clearer.

Reply:
Done